cime_config/tests.py

```diff
@@ -165,10 +165,10 @@
             "SMS_Ld2.ne30_oECv3.BGCEXP_BCRC_CNPRDCTC_1850.elm-bgcexp",
             "SMS_D_Ld1.T62_oEC60to30v3.DTESTM",
             "SMS_D_Ld1.ne30_r05_oECv3.A_WCYCL1850",
-            "ERS_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX",
+            "ERP_Ln9_P96x1.ne4pg2_ne4pg2.F-MMFXX.eam-mmf_max_subcycle",
```

Comment:
I kind of think "fixed_subcycle" might be more appropriate, since it's the fact that the subcycle is normally dynamic that breaks ERP tests, and using a fixed ncycle is what is needed to pass ERP. But I'm fine keeping "max". Just something to consider.

Reply:
good point, I'll change it.

Reply:
Done.